Hspi/html/feature.js

```diff
@@ -0,0 +1,40 @@
+function getUrlParameterOrEmpty(key) {
+	return getUrlParameter(key) ?? '';
+};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/dk307/HSPI_InfluxDBPersistence/pullRequest?prid=8479377)
```suggestion
}
```